dependencies.py

```diff
@@ -47,7 +47,7 @@
     Dependency("threedi-modelchecker", "threedi_modelchecker", ">=0.12"),
     Dependency("threedidepth", "threedidepth", "==0.4"),
     Dependency("click", "click", ">=7.0"),
-    Dependency("alembic", "alembic", ">=0.9"),
+    Dependency("alembic", "alembic", "==1.6.5"),
```

Comment:
Most of the external dependencies aren't pinned *so* hard.
I understand that there's some issue with the latest 1.7.1 version. For that, add a quick note with the reason (or put a note in the changelog and point at the changelog here). 

Alternatively, would a `<=1.7` be nicer? (Perhaps leave that be, though :-) )